documentation.html

```diff
@@ -1508,6 +1587,29 @@ <h1>Made with lots of
     closeBtn.onclick = function () {
         modal.style.display = "none"
     }
+
+    // For Rating
+
+    const allstars = document.querySelectorAll(".star");
+
+    allstars.forEach((star, i) => {
+        star.addEventListener("click", () => {
```

Comment:
Code looks clean and good. Just a suggestion, here you can define the callback function separately and then call it here to keep the code more modular.